pkgs/by-name/sv/svndumpsanitizer/package.nix

```diff
@@ -0,0 +1,26 @@
+{lib, gccStdenv, fetchFromGitHub }:
+
+gccStdenv.mkDerivation rec {
+  pname = "svndumpsanitizer";
+  version = "2.0.7";
+  src = fetchFromGitHub {
+    owner = "dsuni";
+    repo = pname;
+    rev = "cbf917c000ee42fbb31a8667014a4357bbfdd6a6";
+    hash = "sha256-lkVX11t0AF4y1EQQFXjPTrJmsfJhgx5Y1xj1VDlsbH0=";
+  };
+  buildPhase = ''
+    gcc svndumpsanitizer.c -o svndumpsanitizer
+  '';
+  installPhase = ''
+    mkdir -p $out/bin
+    cp svndumpsanitizer $out/bin
+  '';
+  meta = with lib; {
+    description = "Alternative to svndumpfilter that discovers which nodes should actually be kept";
+    homepage = "https://miria.homelinuxserver.org/svndumpsanitizer";
+    license = licenses.gpl3;
```

Comment:
```suggestion
    license = licenses.gpl3Plus;
```

Reply:
Good catch! I don't know how I missed this…